docs/api/menu.md

```diff
@@ -167,102 +174,104 @@ Menu.setApplicationMenu(menu);
 
 ## Class: Menu
 
-### new Menu()
+### `new Menu()`
 
 Creates a new menu.
 
-### Class Method: Menu.setApplicationMenu(menu)
+## Methods
+
+The `menu` class has the following methods:
+
+### `Menu.setApplicationMenu(menu)`
 
 * `menu` Menu
 
 Sets `menu` as the application menu on OS X. On Windows and Linux, the `menu`
 will be set as each window's top menu.
 
-### Class Method: Menu.sendActionToFirstResponder(action)
+### `Menu.sendActionToFirstResponder(action)` _OS X_
 
 * `action` String
 
-Sends the `action` to the first responder of application, this is used for
+Sends the `action` to the first responder of application. This is used for
 emulating default Cocoa menu behaviors, usually you would just use the
 `selector` property of `MenuItem`.
 
-**Note:** This method is OS X only.
-
-### Class Method: Menu.buildFromTemplate(template)
+### `Menu.buildFromTemplate(template)`
 
 * `template` Array
 
-Generally, the `template` is just an array of `options` for constructing
-[MenuItem](menu-item.md), the usage can be referenced above.
+Generally, the `template` is just an array of `options` for constructing a
+[MenuItem](menu-item.md). The usage can be referenced above.
 
-You can also attach other fields to element of the `template`, and they will
-become properties of the constructed menu items.
+You can also attach other fields to the element of the `template` and they
+will become properties of the constructed menu items.
 
-### Menu.popup(browserWindow, [x, y])
+### `Menu.popup(browserWindow[, x, y])`
```

Comment:
@zcbenz I want to make sure I notate this correctly, do `x` and `y` need to be passed in as a pair or as two arguments? Also, do you need _both_ and `x` and `y`?